scalardb-analytics-postgresql-sample/README.md

```diff
@@ -0,0 +1,233 @@
+# A Sample Application of ScalarDB Analytics with PostgreSQL
+
+This pages describe [the sample application](https://github.com/scalar-labs/scalardb-samples/scalardb-analytics-postgresql) of ScalarDB Analytics with PostgreSQL.
+
+## Setting up the database
+
+First, you need to set up the database to run analytical queries with ScalarDB Analytics with PostgreSQL. If you haven't done it yet, please follow the instructions in [the Getting Started page](./getting-started.md).
+
+## Schema in ScalarDB
+
+In this application, you have the tables with the following schema in the ScalarDB database:
+
+```mermaid
+erDiagram
+    "dynamons.customer" ||--|{ "postgresns.orders" : "custkey"
+    "dynamons.customer" {
+      int c_custkey
+      text c_name
+      text c_address
+      int c_nationkey
+      text c_phone
+      double c_acctbal
+      text c_mktsegment
+      text c_comment
+    }
+    "postgresns.orders"  ||--|{ "cassandrans.lineitem" : "orderkey"
+    "postgresns.orders" {
+      int o_orderkey
+      int o_custkey
+      text o_orderstatus
+      double o_totalprice
+      text o_orderdate
+      text o_orderpriority
+      text o_clerk
+      int o_shippriority
+      text o_comment
+    }
+    "cassandrans.lineitem" {
+     int l_orderkey
+     int l_partkey
+     int l_suppkey
+     int l_linenumber
+     double l_quantity
+     double l_extendedprice
+     double l_discount
+     double l_tax
+     text l_returnflag
+     text l_linestatus
+     text l_shipdate
+     text l_commitdate
+     text l_receiptdate
+     text l_shipinstruct
+     text l_shipmode
+     text l_comment
+    }
+```
```

Comment:
I guess other samples show schema in the JSON format because they use `schema-loader` to create tables. The `schema-loader` defines tables in JSON format. But, since this samples use `sql-cli` instead of `schema-loader` to create tables, I think it would be better to show the schema of this sample app in another format.

Reply:
Got it. Thank you for the explanation!